### ISSUE TITLE
Yobit withdraw - [Exchange Error] yobit does not have currency code BTC?

### ISSUE THREAD

**Issue by rsparrow** [ccxt/ccxt#2675]
- OS: OSX
- Programming Language version: JS
- CCXT version: 1.13.46
- Exchange: yobit
- Method: withdraw

This method has the same issue as #2664.

```[Exchange Error] yobit does not have currency code BTC
Error: yobit does not have currency code BTC
    at yobit.currency (.../node_modules/ccxt/js/base/Exchange.js:710:15)
    at yobit.withdraw (.../node_modules/ccxt/js/yobit.js:183:29)

**Comment by kroitor** [comment]
Fixed it as well, in 1.13.48. Thx for reporting!

**Comment by rsparrow** [comment]
Confirmed fixed and working in 1.13.48!
